ajax/libs/san/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "san",
+  "filename": "san-router.min.js",
```

Comment:
I think it should be san.min.js from the dist folder

Reply:
done, thanks!